src/components/DatePicker/story.js

```diff
@@ -14,17 +14,19 @@ storiesOf('Datepicker', module)
   .addDecorator((story, context) => withInfo('Basic datepicker')(story)(context))
   .add('basics', () => (
     <DatePicker
+      error={boolean('error', null)}
+      errorSubInfo={text('errorSubInfo', 'Error!')}
+      icon={<i className="fa fa-calendar" aria-hidden="true" />}
       id="date-picker"
-      name="date-picker"
-      label="Datepicker"
       inputStyle="block"
-      onChange={action('onChange')}
-      onFocus={action('onFocus')}
+      label="Datepicker"
```

Comment:
Could you also add the ```showMonthDropdown``` ```showYearDropdown``` and ``` useShortMonthInDropdown``` handles like -> ```showMonthDropdown={boolean('showMonthDropdown', false}```

Reply:
True! Even more were missing, such as `dateFormat` and `isRequired`...